builtin/credential/github/path_config.go

```diff
@@ -110,8 +126,8 @@ func (b *backend) Config(s logical.Storage) (*config, error) {
 }
 
 type config struct {
-	Org     string        `json:"organization"`
-	BaseURL string        `json:"base_url"`
-	TTL     time.Duration `json:"ttl"`
-	MaxTTL  time.Duration `json:"max_ttl"`
+	Org     string        `json:"organization" structs:"organization" mapstructure:"organization"`
```

Comment:
Actually, the public name of the field is organization: https://www.vaultproject.io/docs/auth/github.html.  Unless I'm misunderstanding what you're suggesting, I think we'd want it to come back in the read with the same name.